### DISCUSSION TITLE
Data Question

### DISCUSSION THREAD

**Discussion by heliumgirl** [davetapley/helium-tax#34] [General]
Is it possible to download data for a date range or do you have to download all transactions each time you utilize?

**Comment by davetapley** [comment]
@heliumgirl right now it's limited to 'all' or a single year.
If there's sufficient interest I could add something like a date picker, but it's unlikely I'll have the time 😁 

You might also look in to [other tools](https://explorer.helium.com/tools), let me know if there's one which does!
